code/modules/surgery/surgery.dm

```diff
@@ -43,7 +43,7 @@
 	if(replaced_by == /datum/surgery)
 		return FALSE
 
-	if(HAS_TRAIT(user, TRAIT_SURGEON))
+	if(istype(user) && user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON))
```

Comment:
```suggestion
	if(user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON))
```
what even is it supposed to do?

Reply:
Truth be told I copied that line from the other check, ill do quick test on your suggestion and then commit it

Reply:
Actuely would just having `if(HAS_TRAIT(user.mind, TRAIT_SURGEON))` not work too? the only problem would be that it woundlt check if there is a mind to check for surgery

Reply:
Ike said having check was better to avoid runtime

Reply:
Probably yes